irc/server.go

```diff
@@ -1412,23 +1413,22 @@ func (server *Server) applyConfig(config *Config, initial bool) error {
 	}
 
 	// set RPL_ISUPPORT
+	var newISupportReplies [][]string
 	oldISupportList := server.isupport
 	server.setISupport()
 	if oldISupportList != nil {
-		newISupportReplies := oldISupportList.GetDifference(server.isupport)
-		// push new info to all of our clients
-		server.clients.ByNickMutex.RLock()
-		for _, sClient := range server.clients.ByNick {
-			for _, tokenline := range newISupportReplies {
-				// ugly trickery ahead
-				sClient.Send(nil, server.name, RPL_ISUPPORT, append([]string{sClient.nick}, tokenline...)...)
-			}
-		}
-		server.clients.ByNickMutex.RUnlock()
+		newISupportReplies = oldISupportList.GetDifference(server.isupport)
 	}
 
 	server.loadMOTD(config.Server.MOTD)
 
+	// reload logging config
+	err = server.logger.ApplyConfig(config.Logging)
+	if err != nil {
+		return err
+	}
+	dumpingRawInOut := server.logger.DumpingRawInOut()
```

Comment:
Hmm, this should be named `nowDumpingRawInOut` and only be true if dumping raw in+out wasn't happening before, and is happening now with the new config